Branch2018/test1/Program.cs

```diff
@@ -0,0 +1,338 @@
+                                        Console.WriteLine("2. Customers older to 55 years.");
+                                        Console.WriteLine("3. Customers who meet years in this month.");
+                                        Console.WriteLine("4. Products low stock.");
+                                        Console.WriteLine("5. Return.");
+                                        Console.WriteLine("Select an option");
+                                        boolAnswer = Int32.TryParse(Console.ReadLine(),out answer);
+                                    }   
+                                        while (boolAnswer == false);                              
+                                    opcion = answer;
+                                    switch (opcion)
+                                    {
+                                        case 1: SalesReport(salesList);
+                                                break; 
+                                                
+                                        case 2: CustomersReport(customersList);
+                                                break;
+                                                
+                                        case 3: CustomerReportAge(customersList);
+                                                break; 
+                                                
+                                        case 4: ProductsReport(productsList);
+                                                break; 
+                                                
+                                        case 5: break;
+                                    }
+                                }
+                            }
+                            break;
+                }
+            }
+            Console.WriteLine("Thank for your visit.");
+        }
+
+        public static void SalesReport(List<Sale> salesList)
+        {
+            bool p = true;
+            Console.WriteLine("Total day sales :"+ salesList.Count);
+            if(salesList.Count == 0)
+            {
+                Console.WriteLine("There are not registered sales.");
+                p = false; 
+            }
+            else
+                {
+                    foreach (Sale sale in salesList)
+                    {
+                            if (p)
+                            {                            
+                                Console.WriteLine("--------------");
+                                Console.WriteLine("Customer code :" + $"{sale.CustomerId},  Product code: {sale.ProductId}, Sales description : {sale.Description}");                               
+                            }
+                    }
+                }
+        }
+
+        public static void CustomersReport(List<Customer> customersList)
+        {
+            bool p = true;
+            Console.WriteLine("Customers older to 55 years:");
+

```

Comment:
this blank line is right, because split 2 portion of code.

Reply:
okey